grid/client.py

```diff
@@ -57,6 +60,20 @@ def _send_http_request(
 
         return response
 
+    def _send_streaming_post(
+        self, route, data=None, N: int = 10, unhexlify: bool = True
+    ):
```

Comment:
Can you add a docstring explaining why this is needed / what this method implements?

Reply:
Done! (b017e19)